openbas-front/src/admin/components/atomic_testings/atomic_testing/AtomicTestingHeader.tsx

```diff
@@ -58,85 +75,116 @@ const AtomicTestingHeader = () => {
   const handleCloseEdit = () => setEdition(false);
 
-  return (
-    <>
-      <Tooltip title={injectResultOverviewOutput.inject_title}>
-        <Typography
-          variant="h1"
-          gutterBottom={true}
-          classes={{ root: classes.title }}
+  function getActionButton(injectResultOverviewOutput: InjectResultOverviewOutput) {
+    if (!injectResultOverviewOutput.inject_injector_contract) {
+      return null;
+    }
+    if (injectResultOverviewOutput.inject_ready && !injectResultOverviewOutput.inject_status?.status_id) {
+      return (
+        <Button
+          style={{ marginRight: 10 }}
+          startIcon={<PlayArrowOutlined />}
+          variant="contained"
+          color="primary"
+          size="small"
+          onClick={handleOpenDialog}
+          disabled={!canLaunch}
         >
-          {truncate(injectResultOverviewOutput.inject_title, 80)}
-        </Typography>
-      </Tooltip>
-      <div className={classes.actions}>
-        {/* eslint-disable-next-line no-nested-ternary */}
-        {injectResultOverviewOutput.inject_injector_contract ? (
-          !injectResultOverviewOutput.inject_ready
-            ? (
-                <>
-                  <Button
-                    style={{ marginRight: 10 }}
-                    startIcon={<SettingsOutlined />}
-                    variant="contained"
-                    color="warning"
-                    size="small"
-                    onClick={handleOpenEdit}
-                  >
-                    {t('Configure')}
-                  </Button>
-                  <AtomicTestingUpdate
-                    open={edition}
-                    handleClose={handleCloseEdit}
-                    atomic={injectResultOverviewOutput}
-                  />
-                </>
-              )
-            : (
-                <Button
-                  style={{ marginRight: 10 }}
-                  startIcon={<PlayArrowOutlined />}
-                  variant="contained"
-                  color="primary"
-                  size="small"
-                  onClick={() => setOpen(true)}
-                  disabled={!availableLaunch}
-                >
-                  {t('Launch')}
-                </Button>
-              )) : null}
-        <AtomicTestingPopover
-          atomic={injectResultOverviewOutput}
-          actions={['Update', 'Duplicate', 'Delete']}
-          onDelete={() => navigate('/admin/atomic_testings')}
-        />
-      </div>
+          {t('Launch now')}
+        </Button>
+      );
+    } else if (injectResultOverviewOutput.inject_ready && injectResultOverviewOutput.inject_status?.status_id) {
+      return (
+        <Button
+          style={{ marginRight: 10 }}
+          startIcon={<PlayArrowOutlined />}
+          variant="contained"
+          color="primary"
+          size="small"
+          onClick={handleOpenDialog}
+          disabled={!canLaunch}
+        >
+          {t('Relaunch now')}
+        </Button>
```

Comment:
Your code contains duplication; the two buttons are identical except for their labels.